functions.php

```diff
@@ -69,33 +69,33 @@ function gesso_scripts() {
   wp_enqueue_script('gessocommon');
 
-  wp_register_script('gessomodernizr', get_template_directory_uri() . '/js/libraries/modernizr.min.js', array('jquery'), $gesso_version ); // Modernizr
+  wp_register_script('gessomodernizr', get_template_directory_uri() . '/js/libraries/modernizr.min.js', array('jquery'), filemtime(get_template_directory_uri() .'/js/libraries/modernizr.min.js' ) ); // Modernizr
   wp_enqueue_script('gessomodernizr');
 
-  wp_register_script('gessodetailspolyfill', get_template_directory_uri() . '/js/libraries/details-element-polyfill.js', array('jquery'), $gesso_version ); // Details polyfill for IE/Edge
+  wp_register_script('gessodetailspolyfill', get_template_directory_uri() . '/js/libraries/details-element-polyfill.js', array('jquery'), filemtime(get_template_directory_uri() . '/js/libraries/details-element-polyfill.js' ) ); // Details polyfill for IE/Edge
   wp_enqueue_script('gessodetailspolyfill');
 
   if ( is_singular() && comments_open() ) {
     wp_enqueue_script( "comment-reply" );
   }
 
-  wp_register_script('gessomobilemenu', get_template_directory_uri() . '/js/dist/mobile-menu.min.js', array('jquery','gessocommon','gessomodernizr'), $gesso_version ); // Mobile menu
+  wp_register_script('gessomobilemenu', get_template_directory_uri() . '/js/dist/mobile-menu.min.js', array('jquery','gessocommon','gessomodernizr'), filemtime(get_template_directory_uri() .  '/js/dist/mobile-menu.min.js' ) ); // Mobile menu
   wp_enqueue_script('gessomobilemenu');
 
-  wp_register_script('gessoprimarymenu', get_template_directory_uri() . '/js/dist/primary-menu.min.js', array('jquery','gessocommon','gessomodernizr'), $gesso_version ); // Primary menu
+  wp_register_script('gessoprimarymenu', get_template_directory_uri() . '/js/dist/primary-menu.min.js', array('jquery','gessocommon','gessomodernizr'), filemtime(get_template_directory_uri() . '/js/dist/primary-menu.min.js' ) ); // Primary menu
   wp_enqueue_script('gessoprimarymenu');
 
-  wp_register_script('gessoscripts', get_template_directory_uri() . '/js/dist/scripts.min.js', array('jquery','gessocommon','gessomodernizr'), $gesso_version ); // Custom scripts
+  wp_register_script('gessoscripts', get_template_directory_uri() . '/js/dist/scripts.min.js', array('jquery','gessocommon','gessomodernizr'), filemtime(get_template_directory_uri() . '/js/dist/scripts.min.js') ); // Custom scripts
   wp_enqueue_script('gessoscripts');
 
   // Add Google Fonts
   wp_register_style('gessoFonts', 'https://fonts.googleapis.com/css?family=Source+Sans+Pro:300,300i,400,400i,600,600i,700,700i&display=swap', $gesso_version );
   wp_enqueue_style( 'gessoFonts');
 
-  wp_enqueue_style( 'style', get_stylesheet_directory_uri() . '/css/styles.css', array(), $gesso_version, 'all' );
+  wp_enqueue_style( 'style', get_stylesheet_directory_uri() . '/css/styles.css', array(), filemtime( get_template_directory_uri() . '/css/styles.css' ), 'all' );
```

Comment:
@sbrack8t this should be using `get_sylesheet_directory()` when referencing stylesheets.

Reply:
Updated